Components/Input/input.css

```diff
@@ -0,0 +1,74 @@
+.input-label {
+    text-align: start;
+    font-size: 1.5rem;
+    font-weight: bolder;
+    color: var(--primary-color);
+}
+
+.input-label.input-textbox {
```

Comment:
Fix the typo it's `input-label, .input-textbox{}`

Reply:
hey! thanks for that.. i have fixed it now. please approve